Shuttle-front/src/app/navbar-module/navbar-module.module.ts

```diff
@@ -0,0 +1,37 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { AdminNavbarComponent } from './admin-navbar/admin-navbar.component';
+import {UnregisteredNavbarComponent} from "./unregistered-navbar/unregistered-navbar.component";
+import { DriverNavbarComponent } from './driver-navbar/driver-navbar.component';
+import { PassengerNavbarComponent } from './passenger-navbar/passenger-navbar.component';
+import {MatToolbarModule} from "@angular/material/toolbar";
+import {MatButtonModule} from "@angular/material/button";
+import {MatSlideToggleModule} from "@angular/material/slide-toggle";
+import {MatIconModule} from "@angular/material/icon";
+import { NavbarComponent } from './navbar/navbar.component';
+
+
+
+@NgModule({
+  declarations: [
+    AdminNavbarComponent,
+    UnregisteredNavbarComponent,
```

Comment:
Why isn't this component in the `exports` array like all other declared components?